modules/imaging_uploader/php/imaging_uploader.class.inc

```diff
@@ -369,6 +369,27 @@ class Imaging_Uploader extends \NDB_Menu_Filter_Form
                 " .tgz, .tar.gz or .zip."
             );
         }
+        ///////////////////////////////////////////////////////////////////////
+        //////// Make sure there's no existing identical entry ////////////////
+        //////// for "new" scans //////////////////////////////////////////////
+        ///////////////////////////////////////////////////////////////////////
+        $updateFile = $args['values']['overwrite'] ? true : false;
```

Comment:
I'm not sure this ternary operator is necessary. What's the value of 'overwrite'? Can you just pass it along to L387?

Reply:
i just copied it over from the `importFile($file, $args)` function. i think it only gets set when the user confirms they want the original upload overwritten